### PR TITLE
Improvement of the documentation

### PR DESCRIPTION
**Pull request by Th3Mouk**
Hello I just found this bundle, what I probably have a try.

## Subject

Improvement of the documentation

## Causes

Some points can be clearer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ public function registerBundles()
 
 ## Commands
 
-Delete expired nonces via the ``escape:wsseauthentication:nonces:delete`` command that ships with this bundle; it takes the firewall name as a (required) parameter.
+Delete expired nonces via the `escape:wsseauthentication:nonces:delete` or alias `escape:w:n:d` command that ships with this bundle; it takes the firewall name as a (required) parameter.
```
Comment (djoos) [inline README.md]: I'd actually leave out the aliased version of the command here... What do you think?

Comment (Th3Mouk) [inline README.md]: It's just an additional information, no collision is possible actually, no code need to be update

Comment (djoos) [inline README.md]: True, but I'd prefer it to be left out... Other than that this is good to go!

Comment (Th3Mouk) [inline README.md]: No problem just let me reclone this repo

Comment (djoos) [inline README.md]: 👍

**Comment by djoos** [comment]
Thanks for your contribution @Th3Mouk - have a great day!

**Comment by Th3Mouk** [comment]
You're welcome ! U too 😊. This bundle is now adopted 👍 I will keep an eye on it

**Comment by djoos** [comment]
Perfect - glad you're up and running!

**Pull request merged**